package.json

```diff
@@ -10,9 +10,16 @@
     "server": "json-server --watch ./server/database.json",
     "lint": "ng lint"
   },
+  "lint-staged": {
```

Comment:
`lint-staged` is listed but also omitted from `devDependencies` in the `package.json` file